src/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>ToDo List</title>
+  </head>
+  <body>
+    <section>
+      <div class="heading">
+        <h1>Today's To Do</h1>
+      </div>
+      <form>
+        <div class="input-row">
+          <input id="task-new" type="text" placeholder="Add to your list" />
+        </div>
+        <ul id="todo-list"></ul>
+        <button type="button">Clear All Completed</button>
+      </form>
+    </section>
+    <ul class="hidden">
+      <li>
+        <input type="checkbox" />
+        <input id="task-edit" type="text" />
+        <img id="more" class="more" src="#" alt="#" />
+        <img id="dele" class="more" src="#" alt="#" />
```

Comment:
Nice job Roberto, so far your code seems great.
I just notice you used an # for the alt, for best practices and accessibility I highly suggest you use a more descriptive image alt.